README.md

```diff
@@ -1,18 +1,20 @@
 # `hardware.inc`
 ### Gameboy Hardware definitions
-`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [rgbds](https://github.com/gbdev/rgbds) projects for over 20 years.
+`hardware.inc` has been the standard include file containing Game Boy hardware definitions for use in [rgbds](https://github.com/gbdev/rgbds) projects for over 20 years. Our intent here at gbdev is to continue to use `hardware.inc` as the standard as it has been for all these years.
 
 The file was originally created by Jeff Frohwein in 1997, who still hosts [his latest version (2.3)](http://www.devrs.com/gb/files/hardware.zip) on his great [Dev'rs](http://devrs.com) website.
 
 Although Jeff tried to version control the file with a rudimentary change log at the top of the file, people have added small changes throughout the years, often without bumping the version number.
 
-This repo is an attempt at version control, using [@AntonioND](http://github.com/AntonioND) fork as baseline.
+This repo is an attempt at version control, using [@AntonioND](http://github.com/AntonioND)'s fork as the baseline.
```

Comment:
Can we call it the new reference, at this point?

Reply:
If this repo is the new reference for `hardware.inc` (not to the GB hardware! That's Pan Docs), then it shouldn't be called "an attempt at version control" anymore. We should however keep that it is based on AntonioND's fork.

Reply:
Edited in latest commit to reflect this